test/unit/range/view/view_single_pass_input_test.cpp

```diff
@@ -193,7 +193,7 @@ TYPED_TEST(single_pass_input, iterator_pre_increment)
     detail::single_pass_input_view view{p};
 
     auto it = view.begin();
-    if constexpr (std::is_same_v<ranges::range_value_type_t<TypeParam>, char>)
+    if constexpr (std::is_same_v<std::ranges::range_value_type_t<TypeParam>, char>)
```

Comment:
@h-2 `std::ranges::range_value_type_t` is still used here (and 3 more times in this file).

Reply:
Then just replace it with `value_type_t` from our namespace.